src/langgraph-platform/deploy-hybrid.mdx

```diff
@@ -50,7 +54,7 @@ The Hybrid deployment option requires an [Enterprise](/langgraph-platform/plans)
         langsmithWorkspaceId: "" # Workspace ID
         hostBackendUrl: "https://api.host.langchain.com" # Only override this if on EU
         smithBackendUrl: "https://api.smith.langchain.com" # Only override this if on EU
-        langgraphListenerId: "" # Listener ID from Step 2-6
+        langgraphListenerId: "" # Listener ID from Step 2f
```

Comment:
The site is now rendering `a-z` for the sub-bullet points (preferred). It was previously `1-9`.